src/main/java/org/cyclonedx/util/LicenseDeserializer.java

```diff
@@ -55,10 +55,19 @@ public LicenseChoice deserialize(
     MapType type = factory.constructMapType(HashMap.class, String.class, License.class);
     LicenseChoice licenseChoice = new LicenseChoice();
     for (JsonNode node : nodes) {
-      HashMap<String, License> map = this.mapper.readValue(node.toString(), type);
-
-      if (map.get("license") != null) {
-        licenseChoice.addLicense(map.get("license"));
+      HashMap<String, License> map = null;
```

Comment:
I think you could do a try-with-resources that might just be a tinier bit more concise:

`try (HashMap<String, License> map = this.mapper.readValue(node.toString(), type)) {`

Everything else is likely the same!

Reply:
The try-with-resources syntax requires that the object being created implements AutoClosable, which HashMap does not and hence the syntax couldn't be used.  The whole idea behind try-with-resources was to streamline the code by removing the need for a finally clause the closes the object.    see: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
You are correct, my head was on autopilot, hah sorry.

Reply:
No worries, been there.